gallery/src/pages/components/ha-alert.ts

```diff
@@ -98,7 +98,7 @@ const alerts: {
     description: "Alert with slotted image",
     type: "warning",
     iconSlot: html`<span slot="icon" class="image"
-      ><img src="https://www.home-assistant.io/images/home-assistant-logo.svg"
+      ><img alt="" src="https://www.home-assistant.io/images/home-assistant-logo.svg"
```

Comment:
I believe it is the responsibility of the surrounding content to provide reasonable context.